airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -105,8 +105,8 @@ class PodEventType(Enum):
     NORMAL = "Normal"
 
 
-class PodReattachFailure(AirflowException):
-    """When we expect to be able to find a pod but cannot."""
```

Comment:
We should return back this exception

Reply:
@Taragolis I've searched for this exception and found that it is not used anywhere anymore. Do you think it is worth keeping?

Reply:
We remove things which not marked as internal thought deprecations, so better just keep it, otherwise it might be considered as Breaking changes (or not) and required to release new major version of provider, and I guess it not worthwhile to release major version of provider.

We could deprecate it by the separate PR and remove it one day, e.g. in the next major version

Reply:
@Taragolis the `PodReattachFailure` exception is back.